tools/internal_ci/windows/grpc_build_artifacts.cfg

```diff
@@ -16,7 +16,7 @@
 
 # Location of the continuous shell script in repository.
 build_file: "grpc/tools/internal_ci/windows/grpc_build_artifacts.bat"
-timeout_mins: 120
+timeout_mins: 180
```

Comment:
I understand this might be necessary but I really really don't like the fact that building the artifacts takes such a long time as it indirectly harms our productivity.
No action needed here but I just wanted to point out this is in a sense technical debt we should be removing at some point.

Reply:
The time for build artifacts are inflating. I agree the time is getting longer, and further investigation into the new Windows image is needed to nail down why the same job took longer time.